data_hub_api/docmaps/v2/sql/docmaps_index.sql

```diff
@@ -367,8 +367,8 @@ t_result_with_sorted_manuscript_versions_array AS (
         result.long_manuscript_identifier,
         result.position_in_overall_stage,
         result.is_or_was_under_review,
-        result.qc_complete_timestamp,
-        result.under_review_timestamp,
+        TIMESTAMP(DATETIME(result.qc_complete_timestamp), 'US/Eastern') AS qc_complete_timestamp,
+        TIMESTAMP(DATETIME(result.under_review_timestamp), 'US/Eastern') AS under_review_timestamp,
```

Comment:
maybe worth mentioning. One hint for me regarding timezone is whether it is coming from a "pretty view". In that case I would usually expect the field spelled `Under_Review_Timestamp`. Not a strong signal I know.
It may also be worth adding a comment?